index.rst

```diff
@@ -202,40 +112,101 @@ We envision that Jupyter Lab servers will be a fundamental part of LSST control
 enabling users to perform low and high level operations in a well-known and
 interactive environment.
 
-The current system deployment uses individual Jupyter Lab servers for each user running out of
-individual docker containers. In the nearby future, the plan is to start using DM LSP
-(:cite:`LDM-542`) environment to manage user servers and environment.
-
-The first step to access a Jupyter Lab server is to tunnel using the IP and token information
-provided for each user by the Telescope & Site team point of contact. Once the ssh tunnel is
-up it should be possible to open a web page on ``localhost:<local-port>``. After entering
-the provided token, you should see the :ref:`Jupyter Lab interface <fig-jupyter-interface>`.
+.. important::
+    As of 2020/01 the old user-based Jupyter Lab server is retired.
+
+For that, users have access to a full-featured `LSST Science Platform instance at the summit`_
```

Comment:
we don't have the full features.... missing all sorts of catalogues and stuff. Maybe call it a "streamlined instance"